### ISSUE TITLE
Rstudio doesn't open but works in the background

### ISSUE THREAD

**Issue by Andrzej-Andrzej** [rstudio/rstudio#10649]
### System details

    RStudio Edition :  Desktop 
    RStudio Version : 2022.02.0 build 428
    OS Version      : Windows 10 x64
    R Version       : 4.1.2/4.1.1

### Steps to reproduce the problem
https://community.rstudio.com/t/newest-dailies-dont-work/129165

### Describe the problem in detail
I can't get RStudio to work after installation, main window with panes are not visible but Rstudio works in the background, that I can check inside windows task manager.

### Describe the behavior you expected
Be able to open main window with panes as usual.

- [x ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@Andrzej-Andrzej Thank you for raising the issue and bringing it over from the RStudio Community! A few questions:

1) Can you try it with the released version, 2022.02.0+443? https://www.rstudio.com/products/rstudio/download/#download
2) Are you using the the 32- or 64-bit version of R? It probably won't make a difference, but will help when trying to reproduce.

**Comment by Andrzej-Andrzej** [comment]
Hi, thank you for your reply. 
1. I can't run stable version 2022.02.0+443. The same effect. This is strange as this is a stable version. The last version from 06 series which is working for me is RStudio-2022.06.0-daily-79. 
2. I am using only x64 R version. 
Maybe this is not relevant but I spotted that these two working in my case versions are around 164 MB in size, meaning - smaller than up-to-date versions of RStudio.
I have got a bit older laptop: 
Sony Vaio, GPU NVidia GT 425M, Windows 10 x 64, Intel Core i5-560M, 8GB Ram DDR3.
